webanno-project/src/main/java/de/tudarmstadt/ukp/clarin/webanno/project/ProjectServiceImpl.java

```diff
@@ -117,6 +117,12 @@ public ProjectServiceImpl()
         // Nothing to do
     }
     
+    public ProjectServiceImpl(EntityManager aEntityManager)
```

Comment:
Please add a comment that this signature is reserved for testing. It would probably be a good idea to take the opportunity to properly use constructor injection on this class...